metriq_gym/metriq_metadata.py

```diff
@@ -95,4 +95,4 @@
     "deutsch_jozsa": 172,
     "quantum_fourier_transform": 142,
     "grovers": 97,
-}
\ No newline at end of file
+}
```

Comment:
Based on the way in which you are handling the upload logic, do we even require this `metriq_metadata.py` file at all? Wondering if we can just remove it.

Reply:
Yes, we do. I use it in this code. In general, we'll need to know task/method/platform IDs as they are stored in the database in order to make API requests. (We could `GET` request to track down the IDs without prior knowledge, but it's extra steps and code, and these primary keys can't change.)